### PR TITLE
Textures optimization and add mod.conf

### PR DESCRIPTION
**Pull request by wsor4035**
copied from the original pr sys4-fr

* Use colorized base texture of white monitor and keyboard for beige
and black variants.
* Remove alpha channel of textures that don't need it.

### CODE REVIEW

**Comment by wsor4035** [comment]
since this has big changes, going to leave this open a bit before merging

File: mod.conf [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = digiterms
+depends = display_api, font_api, digilines, dye, default
+optional_depends = xpane, scifi_nodes, basic_materials, homedecor, mesecons_microcontroller, mesecons_luacontroller, mesecons_button, mesecons_materials
```
Comment (Niklp09) [inline mod.conf]: xpane -> xpanes
homedecor -> homedecor_common ("homedecor" doesn't exist on cdb)

**Review by Niklp09 — APPROVED** [review]
Except things in my comment, lgtm.

File: mod.conf (hunk 1 above)
Comment (wsor4035) [reply mod.conf]: fixed https://github.com/mt-mods/digiterms/pull/1/commits/128fcc47e8cac806d96cd003ac83ff8559771789

**Pull request merged**